test/data/data.ts

```diff
@@ -17,6 +17,153 @@ export const h: ResponseToolkit = {
   }
 }
 
+// @ts-ignore
```

Comment:
ts-ignore is a devil, shouldn't be used....

Reply:
ok. done